packages/custom-webpack/src/custom-webpack-builder-config.ts

```diff
@@ -6,4 +6,10 @@ export interface CustomWebpackBuilderConfig {
   path?: string;
   mergeStrategies?: MergeStrategies;
   replaceDuplicatePlugins?: boolean;
+  verbose?: CustomWebpackBuilderVerboseConfig;
+}
+
+export interface CustomWebpackBuilderVerboseConfig {
+  options?: string | string[];
```

Comment:
What are `options`? I'd prefer the interface to be very explicit with the only allowed properties.

Reply:
Okay.